sematic/ee/README.md

```diff
@@ -0,0 +1,6 @@
+# Alternate License Notice
+
+The code in this directory is only intended for usage with the paid version of
+Sematic. Please see the [LICENSE](./LICENSE) file for reference. Interested in
+using some of the features here? We'd love to hear from you! Reach out on our
+[Discord channel](https://discord.gg/4KZJ6kYVax).
```

Comment:
Not everyone will be interested in using Discord, an email contact is more professional to put here.